.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature Request
+about: Suggest a prompt
+title: "[INSERT PROMPT HERE]"
+labels: enhancement
+assignees: stemaMSFT
+
+---
+
+- [ ] **I have checked the list of issues and believe my issue is not a duplicate.**
+- [ ] **I have read the guide to writing prompts.**
+ 
+**Current output** (optional)
```

Comment:
Do you need to ask for the prompt?

Reply:
@anthony-c-martin what were you thinking as an alternative to asking a prompt?

Reply:
I just meant that I didn't see a section where you were actually asking the user to supply the prompt. Or is the intention that they fill out the section between the `---` at the top?

Reply:
The intention is for the title of the issue itself to be the prompt, thus the title of the issue is "[INSERT PROMPT HERE]"